docs/plugin-v2-logging.md

```diff
@@ -0,0 +1,30 @@
+# Plugin Diagnostic Logging
```

Comment:
I think the filename should be changed to remove the `v2`. The second paragraph already explains that it's related to one of the two plugin models. If we add debugging/logging docs for the other plugin model, this document could be modified, rather than creating a whole new file.

Reply:
> If we add debugging/logging docs for the other plugin model, this document could be modified, rather than creating a whole new file.

That won't happen, but I buy the sentiment :D

Fixed the name